spyder_kernels/console/kernel.py

```diff
@@ -485,6 +486,18 @@ def show_mpl_backend_errors(self):
         if self._mpl_backend_error is not None:
             print(self._mpl_backend_error)  # spyder: test-skip
 
+    def set_sympy_forecolor(self, background_color='dark'):
+        """Set SymPy forecolor depending on console background."""
+        if os.environ.get('SPY_SYMPY_O') == 'True':
+            try:
+                from sympy import init_printing
+                if background_color == 'dark':
+                    init_printing(forecolor='White')
```

Comment:
@ccordoba12 @juanis2112 after testing this with spyder-ide/spyder#10114 the `init_printing` call also needs the [`ip` parameter](https://github.com/sympy/sympy/blob/0ddeeea520a273bb31e672a549529caa2ed91d9c/sympy/interactive/printing.py#L369) (the IPython instance needs to be passed).

For that @juanis2112 please:
* Add the import for `get_ipython` (`from IPython.core.getipython import get_ipython`) below the one for `Sympy`,
* Then use it in the call of `init_printing`, it should look something like `init_printing(forecolor='White', ip=get_ipython())`  (this in each call made).